docs/index.md

```diff
@@ -0,0 +1,14 @@
+# StarkNet Smart Contracts
+
+## Introduction
+
+The climate emergency is the greatest challenge of our time, and requires immediate & widespread action.  
+Blockchain technology is revolutionary and opens up a new world of opportunities.  
+What if we leveraged Blockchain as a tool to fight climate change?  
+
+Our smart contracts were first deployed on Cosmos. To scale up our business we recentely decided to move on StarkNet. A blockchain's sovereignty is a tool that allows a community to generate the conditions needed to solve financial and coordination issues.
+
+- Our first criteria is the energy consumption of the blockchain that we will use. This is why we choose a blockchain with a Proof of Stake consensus.
+- Our second criteria is the maturity and resilience of the blockchain.
```

Comment:
I would change this point in "Our third criteria" and after the first sentence :
`This is why we choose to rely on Ethereum blockchain.`


Because just say "Our second criteria is the maturity and resilience of the blockchain" and Starknet which is still not on mainnet could be see as inconsistency ^^